mrmustard/math/backend_tensorflow.py

```diff
@@ -542,13 +542,7 @@ def hermite_renormalized_diagonal_reorderedAB(
             # The following import must come after running "jl = Julia(compiled_modules=False)" in settings.py
             from julia import Main as Main_julia  # pylint: disable=import-outside-toplevel
 
-            (
-                poly0,
-                poly2,
-                poly1010,
-                poly1001,
-                poly1,
-            ) = Main_julia.DiagonalAmps.fock_diagonal_amps(
+            (poly0, poly2, poly1010, poly1001, poly1,) = Main_julia.DiagonalAmps.fock_diagonal_amps(
```

Comment:
```suggestion
            (poly0, poly2, poly1010, poly1001, poly1) = Main_julia.DiagonalAmps.fock_diagonal_amps(
```
Removing the comma at the end is the way to avoid other reformats of this sort